Utility Mods/Stable/Systems Highlight/Data/Scripts/SystemsHighlight/Systemhighlight.cs

```diff
@@ -80,7 +80,7 @@ public override void BeforeStart()
         public override void LoadData()
         {
             // Return if Server-Side | Only Runs on Client
-            if (MyAPIGateway.Session.IsServer && MyAPIGateway.Utilities.IsDedicated)
+            if (MyAPIGateway.Session.IsServer)
```

Comment:
this seems like it might break in singleplayer/localhost but we aren't that so we good
MyAPIGateway.Utilities.IsDedicated